hooks/use-sound.ts

```diff
@@ -0,0 +1,103 @@
+'use client';
+import type {State} from '@app/config/state';
```

Comment:
There should be a new line between these two declarations.

```suggestion
'use client';

import type {State} from '@app/config/state';
```